Modules/TPC/src/Utility.cxx

```diff
@@ -53,6 +55,36 @@ std::vector<TCanvas*> toVector(std::vector<std::unique_ptr<TCanvas>>& input)
   return output;
 }
 
+void fillCanvases(const o2::tpc::CalDet<float>& calDet, std::vector<std::unique_ptr<TCanvas>>& canvases, const std::unordered_map<std::string, std::string>& params, const std::string paramName)
+{
+  const std::string parNBins = paramName + "NBins";
+  const std::string parXMin = paramName + "XMin";
+  const std::string parXmax = paramName + "XMax";
+  int nbins = 300;
+  float xmin = 0;
+  float xmax = 0;
+  const auto last = params.end();
+  const auto itNbins = params.find(parNBins);
+  const auto itXmin = params.find(parXMin);
+  const auto itXmax = params.find(parXmax);
+  if ((itNbins == last) || (itXmin == last) || (itXmax == last)) {
+    LOGP(warning, "missing parameter {}, {} or {}, falling back to auto scaling", parNBins, parXMin, parXmax);
+  } else {
+    nbins = std::stoi(itNbins->second);
+    xmin = std::stof(itXmin->second);
+    xmax = std::stof(itXmax->second);
```

Comment:
Make naming consistent
```suggestion
  const std::string parXMax = paramName + "XMax";
  int nbins = 300;
  float xmin = 0;
  float xmax = 0;
  const auto last = params.end();
  const auto itNBins = params.find(parNBins);
  const auto itXMin = params.find(parXMin);
  const auto itXMax = params.find(parXMax);
  if ((itNBins == last) || (itXMin == last) || (itXMax == last)) {
    LOGP(warning, "missing parameter {}, {} or {}, falling back to auto scaling", parNBins, parXMin, parXMax);
  } else {
    nbins = std::stoi(itNBins->second);
    xmin = std::stof(itXMin->second);
    xmax = std::stof(itXMax->second);
```
Please also extend the warning message about how to fix this in the json file